source/fuzz/fuzzer_pass_permute_blocks.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/fuzzer_pass_permute_blocks.h"
+
+#include "source/fuzz/transformation_move_block_down.h"
+
+namespace spvtools {
+namespace fuzz {
+
+using opt::IRContext;
```

Comment:
nit: remove, always fully qualified.

Reply:
Ah OK - I thought it was alright to do this in a .cpp file.  Will fully qualify.

Reply:
(Turns out I wasn't even relying on having that using statement...)